DeathRoll/Configuration.cs

```diff
@@ -44,6 +44,7 @@ public class Configuration : IPluginConfiguration
     public bool AutoDrawCard = true;
     public bool AutoDrawOpening = true;
     public bool AutoDrawDealer = true;
+    public bool DealerDrawsAll = true;
```

Comment:
We don't force new ways on the user, so we set it to false by default

Reply:
Fixed!